ambari/src/main/java/io/brooklyn/ambari/AmbariClusterImpl.java

```diff
@@ -95,6 +95,11 @@ public void init() {
         super.init();
 
         setAttribute(AMBARI_SERVER, addChild(createServerSpec(getConfig(SECURITY_GROUP))));
+        if(!getConfig(SERVER_COMPONENTS).isEmpty()) {
+            getAttribute(AMBARI_SERVER).config().set(SoftwareProcess.CHILDREN_STARTABLE_MODE, SoftwareProcess.ChildStartableMode.BACKGROUND_LATE);
```

Comment:
Not sure but I think this can be changed by `Entities.manage(childEntity)`, to be called after the `Entity childEntity = getAttribute(AMBARI_SERVER).addChild(agentSpec);`